tests/test_logger.py

```diff
@@ -111,7 +111,7 @@ def test_an_error_is_raised_when_custom_logging_config_file_does_not_exist():
     configuration occurs."""
     with pytest.raises(RuntimeError) as ex_info:
         _configure_vllm_root_logger()
-    assert ex_info.type == RuntimeError
```

Comment:
There are some cases where we intentionally check the exact types match. Can you ignore all existing cases so they aren't modified by ruff?

Reply:
wouldn't isinstance does this?

Reply:
No, `isinstance` is less strict as subclasses would also match.

Reply:
Also, this change causes `assert_nested_tensors_equal` to fail as shown in the CI logs.